docs/source/traditional-ml/serving-multiple-models-with-pyfunc/notebooks/MME_Tutorial.ipynb

```diff
@@ -0,0 +1,863 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "2877aa82-8820-4685-b646-fb475f4f9af6",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Deploy an MLflow `pyfunc` model with Model Serving\n",
+    "\n",
+    "In this notebook, learn how to deploy MLflow `pyfunc` to a serving endpoint. MLflow pyfunc offers greater flexibility and customization to your deployment. You can run any custom model, add preprocessing or post-processing logic, or execute any arbitrary Python code. While using the MLflow built-in flavor is recommended for optimal performance, you can use MLflow `pyfunc` where more customization is required. The following demonstrates how to deploy a GPT-2 model with `pyfunc` to a GPU endpoint, but the workflow outlined below also works with various other types of models to either CPU or GPU endpoints."
```

Comment:
I feel like it's talking about PythonModels, MLFlow `pyfunc` is quite confusing 🤔